library/ssl_misc.h

```diff
@@ -2773,8 +2773,6 @@ int mbedtls_ssl_session_set_hostname(mbedtls_ssl_session *session,
 
 #if defined(MBEDTLS_SSL_PROTO_TLS1_3) && defined(MBEDTLS_SSL_SESSION_TICKETS)
 
-#define MBEDTLS_SSL_TLS1_3_MAX_ALLOWED_TICKET_LIFETIME (604800)
```

Comment:
Why have you removed it? It was a good improvement.

Reply:
misunderstand your feedback